hack/install/install-runc.sh

```diff
@@ -1,37 +0,0 @@
-#!/bin/bash
```

Comment:
This file is used in contrib/Dockerfile.test; https://github.com/containerd/containerd/blob/master/contrib/Dockerfile.test#L36

Are some of the other scripts useful for local development/testing?

Reply:
Hmm, actually, looks like that Dockerfile may be broken because things moved around? (not sure, will have to check)

Reply:
That one is `script/setup/install-runc`, not `hack/install/install-runc.sh`

Reply:
Ah, 🤦 thought they were renamed or so. thx